client/controllers/curatorEvents.coffee

```diff
@@ -16,6 +16,7 @@ Template.curatorEvents.onCreated ->
     ).map((article)->
       [article.userEventId, article]
     ))
+  Template.curatorEvents.associatedEventIdsToArticles = @associatedEventIdsToArticles
```

Comment:
Please provide an example of passing a reactive var to another template through a helper.